app/models/manageiq/providers/native_crud.rb

```diff
@@ -0,0 +1,129 @@
+class ManageIQ::Providers::NativeCrud < Job
```

Comment:
Yeah I like `Workflow`, aligns better with the `miq_provision_workflow`/`miq_request_workflow`